go.mod

```diff
@@ -109,7 +109,7 @@ require (
 	google.golang.org/grpc v1.27.0
 	gopkg.in/gcfg.v1 v1.2.0
 	gopkg.in/square/go-jose.v2 v2.2.2
-	gopkg.in/yaml.v2 v2.2.8
+	gopkg.in/yaml.v2 v2.3.0
```

Comment:
running `hack/lint-dependencies.sh` will indicate this needs to be updated and vendor updated

Reply:
looks like this version changed the default line width, so we may need to explicitly set the previous default of 80 in our yaml serializers

Reply:
This is the related change https://github.com/go-yaml/yaml/pull/571 but I'm not sure where to start digging in on our end outside of seeing that the tests do.

Reply:
And there are the failures. I'll dig in.

https://prow.k8s.io/view/gs/kubernetes-jenkins/pr-logs/pull/95571/pull-kubernetes-bazel-test/1316503504432402432